dayinfo.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+from brutal.core.plugin import cmd
+from datetime import datetime
+import calendar_sk
+
+
+def formatter(data, month, day, prefix=''):
+    """Format data"""
+    if isinstance(data, list):
+        info = data[month-1][day-1]
```

Comment:
@Adman that's both strange, confusing and wrong.

Let's hear what @pepol has to say about it.